src/overlays/actors/ovl_Dm_Tsg/z_dm_tsg.h

```diff
@@ -5,9 +5,18 @@
 
 struct DmTsg;
 
+typedef void (*DmTsgActionFunc)(struct DmTsg*, PlayState*);
+
 typedef struct DmTsg {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x7DC];
+    /* 0x144 */ DmTsgActionFunc ActionFunc;
```

Comment:
This doesn't seem to be used, why do you think it is an actionfunc?

Reply:
It was a bad guess at the beginning i forgot to remove, thanks.